testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerEndpointTest.java

```diff
@@ -335,6 +335,7 @@ public void testRequestCredentialUnsupportedCredential() throws Throwable {
     @Test
     public void testRequestCredential() {
```

Comment:
How about add these additional check in `requestOffer` ?

Reply:
I agree, thank you:) Fixed in [bf62374](https://github.com/keycloak/keycloak/pull/29806/commits/bf6237402e5039bc326c730eba6d4c7fcf1967b9)

Reply:
I confirmed the fix. Thank you.